monai/apps/pathology/transforms/post/array.py

```diff
@@ -159,37 +179,18 @@ class GenerateInstanceBorder(Transform):
 
     backend = [TransformBackends.NUMPY]
 
-    def __init__(
-        self,
-        kernel_size: int = 21,
-        min_size: int = 10,
-        remove_small_objects: bool = True,
-        dtype: DtypeLike = np.float32,
-    ) -> None:
-
+    def __init__(self, kernel_size: int = 5, dtype: DtypeLike = np.float32) -> None:
         self.dtype = dtype
-
         self.sobel_gradient = SobelGradients(kernel_size=kernel_size)
-        if remove_small_objects:
-            self.remove_small_objects = RemoveSmallObjects(min_size=min_size)
-        else:
-            self.remove_small_objects = None  # type: ignore
 
     def __call__(self, mask: NdarrayOrTensor, hover_map: NdarrayOrTensor) -> NdarrayOrTensor:  # type: ignore
         """
         Args:
-            mask: binarized segmentation result.  Shape must be [1, H, W].
+            mask: binary segmentation map, the output of :py:class:`GenerateWatershedMask`.
+                Shape must be [1, H, W].
```

Comment:
Very minor, but if [H,W] is supplied, could we not add the 1 dimension? Just to be a bit more tolerant.